packages/backend/src/utils/catalogUtils.ts

```diff
@@ -50,7 +50,9 @@ export function sanitize(rawObject: object): ApplicationCatalog {
 }
 
 export function hasCatalogWrongFormat(raw: object): boolean {
-  return 'recipes' in raw && Array.isArray(raw.recipes) && !!raw.recipes.find(r => 'models' in r);
+  return (
+    !('version' in raw) || ('recipes' in raw && Array.isArray(raw.recipes) && raw.recipes.find(r => 'models' in r))
```

Comment:
```suggestion
    !('version' in raw) || ('recipes' in raw && Array.isArray(raw.recipes) && !!raw.recipes.find(r => 'models' in r))
```
i think it's still better to keep the !! to convert it to boolean explicitly

Reply:
Or use `includes` to have a boolean